src/op_handlers/precompile_call.rs

```diff
@@ -73,7 +74,8 @@ pub fn precompile_call(vm: &mut VMState, opcode: &Opcode) -> Result<(), EraVmErr
         }
     }
 
-    vm.set_register(1, TaggedValue::new_raw_integer(1.into()));
+    //vm.set_register(opcode.dst0_index, TaggedValue::new_raw_integer(1.into()));
+    address_operands_store(vm, opcode, TaggedValue::new_raw_integer(1.into()))?;
```

Comment:
The result is not stored on register 1 but on dst0.
This is probably a misinterpretation of `Register1` on vm2